agent/src/beerocks/fronthaul_manager/fronthaul_manager_main.cpp

```diff
@@ -178,15 +178,19 @@ int main(int argc, char *argv[])
         }
     }
 
+    auto syslog = (beerocks_slave_conf.sLog.syslog_enabled == "true");
+
     // Init logger ap_manager
-    std::string base_ap_manager_name = std::string(BEEROCKS_AP_MANAGER) + "_" + fronthaul_iface;
+    std::string base_ap_manager_name =
+        std::string(syslog ? BEEROCKS_FRONTHAUL : BEEROCKS_AP_MANAGER) + "_" + fronthaul_iface;
```

Comment:
This smells like a serious break of abstraction... Could this condition somehow be moved into init_logger itself?

Reply:
No, I wish...
init_logger is just a static function I created in this file. I still would have to provide the two options so it will be kind of the same.